### DISCUSSION TITLE
Tailwind and Nuxt 3.0

### DISCUSSION THREAD

**Discussion by heitkrr** [tailwindlabs/tailwindcss#10260] [Help]
I can't implement Nuxt 3.0 with Tailwind following the [Tailwind documentation](https://tailwindcss.com/docs/guides/nuxtjs#3). Can somebody explain what I'm doing wrong? 

`Cannot find name 'defineNuxtConfig'.`

![Screenshot (2)](https://user-images.githubusercontent.com/69590894/211041576-e5155735-c4fa-422e-93c8-fc0bb7775939.png)

I'm using the Vue.volar extension and take over mode since the [Nuxt 3.0 recommends](https://nuxt.com/docs/getting-started/installation).

**Comment by Mikaleb** [comment]
Double on this

**Reply by heitkrr** [reply]
I will send them a bug request, didn't find the answer yet
